pkg/graveler/graveler.go

```diff
@@ -1198,6 +1200,10 @@ func (g *Graveler) Commit(ctx context.Context, repositoryID RepositoryID, branch
 
 		// fill commit information - use for pre-commit and after adding the commit information used by commit
 		commit = NewCommit()
+
+		if params.Date != nil {
+			commit.CreationDate = time.Unix(*params.Date, 0)
```

Comment:
If date is not provided it will be set to 0?

Reply:
No, if the date is provided ( != nil) it will set the creation date. the second value in `time.Unix(*params.Date, 0)` is the nanoseconds